### PR TITLE
Fixes bug where delimiter applied to header arguments

### PR DESCRIPTION
**Pull request by zcstarr**
Small fix that changes the logic from using the delimiter specified delimiter to parse header arguments.

https://github.com/attic-labs/noms/issues/2456

### CODE REVIEW

**Comment by zcstarr** [comment]
@aboodman  PTAL

**Comment by aboodman** [comment]
test :). You could probably add it to either read_test.go or importer_test.go.

File: samples/go/csv/csv-import/importer.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func main() {
 		headers, err = cr.Read()
 		d.PanicIfError(err)
 	} else {
-		headers = strings.Split(*header, string(comma))
+		headers = strings.Split(*header, string(","))
```
Comment (aboodman) [inline samples/go/csv/csv-import/importer.go]: Is the `string()` call needed?

**Comment by zcstarr** [comment]
@aboodman  PTAL

File: samples/go/csv/csv-import/importer.go [hunk 2]
```diff
@@ -114,7 +114,7 @@ func main() {
 		r = progressreader.New(r, getStatusPrinter(size))
 	}
 
-	comma, err := csv.StringToRune(*delimiter)
+	parsedDelimiter, err := csv.StringToRune(*delimiter)
```
Comment (aboodman) [inline samples/go/csv/csv-import/importer.go]: `parsed` seems like unnecessary information. Can we just use `delim`?

**Comment by aboodman** [comment]
LGTM w/ nit -- just change it and land when ready, no need for another review

**Pull request merged**